### ISSUE TITLE
imagemin warning

### ISSUE THREAD

**Issue by jhnlsn** [yeoman/generator-webapp_DEPRECATED#421]
Unable to process the image min task when there is any image in the images folder.

``` javascript
Running "imagemin:dist" (imagemin) task
Warning: Bad argument Use --force to continue.

Aborted due to warnings.
```

task definition

``` javascript
imagemin: {
            dist: {
                files: [{
                    expand: true,
                    cwd: '<%= config.app %>/images',
                    src: '{,*/}*.{gif,jpeg,jpg,png}',
                    dest: '<%= config.dist %>/images'
                }]
            }
        }
```

**Comment by jhnlsn** [comment]
Verbose output

``` javascript
Running "imagemin:dist" (imagemin) task
Verifying property imagemin.dist exists in config...OK
Files: app/images/logo.png -> dist/images/logo.png
Options: optimizationLevel=7, progressive
Warning: Bad argument Use --force to continue.

Aborted due to warnings.
```

**Comment by jhnlsn** [comment]
it looks like this is related to the fact that 0.6.0 is being used, please upgrade webapp to use imagemin 0.7.0

https://github.com/gruntjs/grunt-contrib-imagemin/issues/218

**Comment by jhnlsn** [comment]
apparently I have no idea what i'm doing with yeoman, it appears this has already been done in the package.json template, however, when I installed webapp I got the 0.6.0 version of imagemin in my package.json file.

**Comment by jhnlsn** [comment]
sorry guys, maybe i've just had too much coffee this morning.  I see whats up now.  the webapp generator's pre-release has the fix but the 0.4.9 version does not.  If you are seeing this error and you installed the generator-webapp, like the instructions say, just update your package.json file to this:

```
"grunt-contrib-imagemin": "~0.7.0",
```
